ant-global.xml

```diff
@@ -38,7 +38,15 @@
             <fileset dir="${ivy.jar.dir}" includes="*.jar"/>
         </path>
         <taskdef resource="org/apache/ivy/ant/antlib.xml" uri="antlib:org.apache.ivy.ant" classpathref="ivy.lib.path"/>
-        <ivy:settings id="dev.settings" file="../zm-zcs/ivysettings.xml"/>
+        <if>
+          <available file="../zm-network-build" type="dir"/>
+          <then>
+            <ivy:settings id="dev.settings" file="../zm-zcs/ivysettings-network.xml"/>
```

Comment:
That is an excellent point.  It made sense at the time that Prashant and I were talking about it.  I will relocate the file and update the two PRs.